### ISSUE TITLE
Errors while starting job : JenkinsApi::Exceptions::ApiException

### ISSUE THREAD

**Issue by algorys** [jbox-web/redmine_jenkins#45]
Hello,

I've an issue with your plugin. When I attempt to make a build with the Jenkins'Tab, I receiver this following error :

```
Errors while starting job 'core_GCC' : JenkinsApi::Exceptions::ApiException
```

I've test connection with the plugin settings and it works. Other fonctionnality of plugin works too (latest build, State, ...).

Is there something to install on the Jenkins side ?

Logins on my Jenkins server is with Unix Account.

**Configuration**
Jenkins 1.620 (Debian)
Redmine 3.0.3 (Sources) (Debian)
Redmine_Jenkins 1.0.1

**Others Plugins :**
Advanced Roadmap : 2.4.2
Redmine Bootstrap 0.2.3
DMSF 1.5.2
Redmine Git Remote 0.0.1
Redmine LDAP Sync 2.0.7Devel
My Page customization 0.1.5
Revision Branches 0.3.0

Thanks for help

**Comment by n-rodriguez** [comment]
> Is there something to install on the Jenkins side ?

Nope. Except the config, it should work out of the box. Maybe the password have special characters?

**Comment by algorys** [comment]
I think that's because I've parameters... I'll make another tests tomorrow

**Comment by n-rodriguez** [comment]
Hi there! Any news?
